packages/block-editor/src/store/selectors.js

```diff
@@ -1623,3 +1623,26 @@ export function isNavigationMode( state ) {
 export function didAutomaticChange( state ) {
 	return !! state.automaticChangeStatus;
 }
+
+/**
+ * Returns the editor settings colors.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {Array} The editor settings colors array.
+ */
+export function getColors( state ) {
+	return get( getSettings( state ), 'colors', [] );
+}
+
+/**
+ * Returns the color object matching the slug, or undefined.
+ *
+ * @param {Object} state Editor state.
+ * @param {string} colorSlug  A string containing the color slug.
+ *
+ * @return {Object} Color object included in the editor settings colors, or Undefined.
+ */
+export const getColorObjectByColorSlug = ( state, colorSlug ) => {
```

Comment:
is there any specific naming pattern you've been following? if the rule is not strictly defined, I would simplify to `getColorBySlug` and keep all the docs and variable names the same to keep the explanation there.

Reply:
I've seen some helper functions in the utils.js file of the colors, for instance [getColorObjectByAttributeValues](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/colors/utils.js#L19).
It doesn't mean that we can't rename it.